### PR TITLE
Bring back carryall.starport and fix trike.starport's prerequisites

### LINKED ISSUE #7769: Carryall is missing from the starport
There is no `carryall.starport` actor defined anywhere any more, so you can't order a carryall from the starport.
Regression from < nobody_really_cares_what >.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/10177213-carryall-is-missing-from-the-starport?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by penev92**
Fixes #7769.

### CODE REVIEW

File: mods/d2k/rules/starport.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ trike.starport:
 	Inherits: trike
 	Buildable:
 		Queue: Starport
+		Prerequisites: starport
```
Comment (obrakmann) [inline mods/d2k/rules/starport.yaml]: This isn't necessary. Only the starport provides the starport queue, so adding it as a prerequisite is redundant.

Comment (penev92) [inline mods/d2k/rules/starport.yaml]: Not really. 
Without this the trike doesn't show up in the starport queue.

Comment (obrakmann) [inline mods/d2k/rules/starport.yaml]: Oh, you're right. Guess I shouldn't have checked in debug mode :s

**Comment by obrakmann** [comment]
:+1:

**Comment by reaperrr** [comment]
:+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/e7371c2309dfd6bed27663dcd51555268855dbd5%5E...e7371c2309dfd6bed27663dcd51555268855dbd5)

**Pull request merged**